fastlane_bot/bot.py

```diff
@@ -817,7 +823,7 @@ def _handle_trade_instructions(
             self.ConfigObj.logger.info(
                 f"[bot._handle_trade_instructions] Opportunity with profit: {num_format(best_profit_gastkn)} does not meet minimum profit: {self.ConfigObj.DEFAULT_MIN_PROFIT_GAS_TOKEN}, discarding."
             )
-            return None, None
+            return None, None, None
```

Comment:
The log_dict is returned alongside the txhash and tx_reciept in the txhelpers.
@barakman can you verify this is correct